### ISSUE TITLE
[Bug] conda install open3d需要GLIBC2.27，Ubuntu16.04不支持

### ISSUE THREAD

**Issue by ranpin** [open-mmlab/mmdetection3d#2588]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection3d/issues) and [Discussions](https://github.com/open-mmlab/mmdetection3d/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection3d.readthedocs.io/en/latest/notes/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (dev-1.x)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x) or [latest version (dev-1.0)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.0).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

1.x branch https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x

### Environment

As the question, the environment configured according to the readme is reporting an error during inference verification. According to the solution searched in issuse, I replaced open3d with version 0.9.0.0, but this does not work because python 3.8 does not support replacing open3d with version 0.9.0.0. How can I solve it?

### Reproduces the problem - code sample

As the question, the environment configured according to the readme is reporting an error during inference verification. According to the solution searched in issuse, I replaced open3d with version 0.9.0.0, but this does not work because python 3.8 does not support replacing open3d with version 0.9.0.0. How can I solve it?

### Reproduces the problem - command or script

As the question, the environment configured according to the readme is reporting an error during inference verification. According to the solution searched in issuse, I replaced open3d with version 0.9.0.0, but this does not work because python 3.8 does not support replacing open3d with version 0.9.0.0. How can I solve it?

### Reproduces the problem - error message

As the question, the environment configured according to the readme is reporting an error during inference verification. According to the solution searched in issuse, I replaced open3d with version 0.9.0.0, but this does not work because python 3.8 does not support replacing open3d with version 0.9.0.0. How can I solve it?

### Additional information

As the question, the environment configured according to the readme is reporting an error during inference verification. According to the solution searched in issuse, I replaced open3d with version 0.9.0.0, but this does not work because python 3.8 does not support replacing open3d with version 0.9.0.0. How can I solve it?

**Comment by JingweiZhang12** [comment]
Why not use open3d of the higher version?

**Comment by ranpin** [comment]
> 

1. I followed the official installation instructions of mmdet3d and the default installation is python=3.8 and open3d=0.13.0, but this requires glibc=2.27, which is not supported by Ubuntu 16.04, who only has glibc=2.23 at the highest.

2.  I searched the issues in mmdet3d and the solution mentioned is to open3d down to 0.9.0 (and the issue #331  says you have fixed this problem, but I still get the error). But  if I manually replace open3d with 0.9.0 it conflicts with python 3.8, which only supports open3d=0.10.0 as a minimum.

It's not that I don't want to use a higher version of open3d, but Ubuntu 16.04 doesn't support it, and using a lower version will conflict with the required python version according to your solution (and the python version can't be lowered, otherwise the installation will also error out). The above two problems, I can not solve, because they are completely contradictory.

**Comment by JingweiZhang12** [comment]
@ranpin Two possible solutions:
1. A tricky solution. I guess the `GLIBC2.27` is required by some dynamic link libs, like `.so` file. Thus, you could create a soft link for this high-version dynamic link lib to some similar low-version dynamic link libs. 
2. You could try python3.7 and install the corresponding compatible libs.

**Comment by ranpin** [comment]
> 
Thanks! I tried to solve the problem from the above two perspectives.
1. For method 1, Since some underlying operations are involved and I don't have administrative privileges, the operation is too tedious and difficult for me.
2. For method 2,  I tried to change to python 3.7 version, in "pip install -v -e ." There are a lot of error warnings in this step, but we can ignore them for now. But when using the default version of open3d=0.13.0, he still reported an error that GLIBC_2.27 is needed. when I refer to the above mentioned open3d to 0.9.0 version, the error will be reported as follows.

![image](https://github.com/open-mmlab/mmdetection3d/assets/51260021/811dfeb1-9cf4-4b10-9036-b4ba9f89e5bc)

To sum up, the main problem is still the python version, open3D version and GLIBC version compatibility issues. It is difficult for me to solve this problem, so I will try to contact the administrator to upgrade GLIBC_2.27 version. Thank you for your answer.


> @ranpin Two possible solutions:
> 
> 1. A tricky solution. I guess the `GLIBC2.27` is required by some dynamic link libs, like `.so` file. Thus, you could create a soft link for this high-version dynamic link lib to some similar low-version dynamic link libs.
> 2. You could try python3.7 and install the corresponding compatible libs.
